packages/features/steps/github-actions.steps.ts

```diff
@@ -0,0 +1,29 @@
+import {exec} from '../lib/system'
+import {When, Then} from '@cucumber/cucumber'
+import * as path from 'pathe'
+import {strict as assert} from 'assert'
+
+When(/I look at the github actions we use/, async function () {
+  const {stdout} = await exec('grep', ['-Roh', 'uses: \\S*', '.'], {cwd: path.join(__dirname, '../../../.github')})
```

Comment:
Is it safe assuming that `grep` is in all the supported OSs?

Reply:
yup, the windows environment on github actions has `/usr/bin/grep`, it's pretty neat